techniques/general/G216.html

```diff
@@ -35,36 +35,36 @@ <h2>達成方法に関する重要な情報</h2>
       <main>
          <section id="applicability">
             <h2>適用 (対象)</h2>
-            <p>ポインタ入力をサポートする全ての技術 (例えば、以下のいずれかまたは全てをサポートしている。マウスポインタ、タッチスクリーン又はトラックパッド上でのタッチ、スタイラス入力、又はレーザーポインタ入力)。
+            <p>ポインタ入力をサポートする全ての技術 (例えば、マウスポインタ、タッチスクリーン又はトラックパッド上でのタッチ、スタイラス入力、又はレーザーポインタ入力のいずれか又は全てをサポートしている)。
```

Comment:
「例えば」があるので、デバイスの並列は「、」だけでつないでみました。そうは言っても「タッチスクリーン又はトラックパッド上」というのはどうしようもない（？）ので、ここはママとします。

```suggestion
            <p>ポインタ入力をサポートする全ての技術 (例えば、マウスポインタ、タッチスクリーン又はトラックパッド上でのタッチ、スタイラス入力、レーザーポインタ入力のいずれか又は全てをサポートしている)。
```

そうでなくて、「スタイラス入力、又はレーザーポインタ入力」をいかすなら、「タッチスクリーンもしくはトラックパッド上」とします。

Reply:
たしかに又はの連続で読みづらくなってしまっていますね。ありがとうございます！